server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/mdc/LoggingResponseFilter.java

```diff
@@ -36,12 +34,9 @@
 
     @Override
     public void handle(Request request, Response response) throws Exception {
-        try (Closeable closeable =
-                 MDCBuilder.create()
-                     .addContext(STATUS, response.status())
-                     .addContext(BODY, response.body())
-                     .build()) {
-            LOGGER.info("Received request");
-        }
+        MDCStructuredLogger.forLogger(LOGGER)
+            .addField(STATUS, response.status())
+            .addField(BODY, request.body())
+            .log(logger -> logger.info("WebAdmin request received"));
```

Comment:
I don't understand this piece of code. You are in Logging*Response*Filter. So you get a request or a response?

Reply:
A response, obviously...

Reply:
so please change the message :)